tests/testthat/test-single_value.R

```diff
@@ -0,0 +1,17 @@
+test_that("single_value", {
+  expect_equal(single_value(1), 1)
+  expect_equal(single_value(c(1, NA)), 1)
+  expect_equal(single_value(c(1, NA, 1)), 1)
+  expect_equal(single_value(NA_real_), NA)
+  expect_equal(single_value(NA, missing=NA_real_), NA_real_)
```

Comment:
Want to add a test for multiple values of missing where the first takes precedence?  E.g., `single_value(c(""), missing = c(NA, ""))` correctly returns NA.

Reply:
Good call.  Added.